tutorials/deploy-ubuntu-20-04-instance-scaleway-elements/index.mdx

```diff
@@ -0,0 +1,127 @@
+---
+meta:
+  title: How to deploy a Ubuntu 20.04 LTS (Focal Fossa) Instance on Scaleway Elements
+  description: In this tutorial you will learn how to deploy, upadate and manage a Ubuntu 20.04 LTS (Focal Fossa) Instance on Scaleway Elements.
+content:
+  h1: How to deploy a Ubuntu 20.04 LTS (Focal Fossa) Instance on Scaleway Elements
+  paragraph: In this tutorial you will learn how to deploy a Ubuntu 20.04 LTS (Focal Fossa) Instance on Scaleway Elements.
+tags: Ubuntu instance setup server 
+  - compute
+dates:
+  validation: 2021-12-02
+  posted: 2021-12-02
+---
+
+[Ubuntu](https://ubuntu.com/) is one of the most popular Linux distributions. 
+First released in 2004, Ubuntu whose name is based on an ancient African word meaning ‘humanity to others’, became quickly the favourite distributon for users around the world, mostly because it is easy to install and use. 
+
+Ubuntu is developed and maintained by the company [Canonical](https://canonical.com/) and a [large community](https://loco.ubuntu.com/). They commercial and community teams release new versions of the distribution every six months and collaborate to produce a sigle, high quality relase with long term support (LTS) every two years.
+
+In this tutorial you will learn how to deploy an Instance with Ubuntu 20.04 LTS (Focal Fossa) from the Scaleway console and configure it so you can host your apps and websites on it. 
+
+<Message type="requirement">
+
+- You have an account and are logged into the [Scaleway console](https://console.scaleway.com)
+- You have [registered a valid payment method](/console/my-account/how-to/manage-billing-information/)
```

Comment:
I don't remember us putting this in the requirements before, though I guess it is true and makes me wonder if it should also be in some of our other tutorials?

Reply:
```suggestion
```